### DISCUSSION TITLE
Attach data to trigger out

### DISCUSSION THREAD

**Discussion by wirmachenbunt** [cables-gl/cables_docs#926] [Answers & Questions] [answered]
is it possible to attach data, let's say an object or a string, to trigger out of an op, and access that data with the next op, connected this trigger ?

if the question is not clear, i want to create a Rapier world object and have another op, working with that world object. do i have to output this world object via an object out port or can i attach this object to the trigger output and access this with another op, receiving this trigger. 

just asking in terms of making less connections between ops

**Comment by pandrr** [comment, marked as answer]
hi

you can use op.patch.cgl.frameStore for this.

e.g. see ammoworld op:

`
    const old = cgl.frameStore.ammoWorld;
    cgl.frameStore.ammoWorld = ammoWorld;

    ...

    cgl.frameStore.ammoWorld = old;
`

**Reply by wirmachenbunt** [reply]
hehe, i've seen that op.patch.cgl.frameStore and guessed that this is doing it, but wasn't sure because of the naming. what stand cgl for ? cables graphics library ? is there a documentation somewhere regarding things like that ?

thanks for the hint
